cds_ils/patrons/views.py

```diff
@@ -69,12 +69,11 @@ def get(self, patron, **kwargs):
         ]
         active_loans = search_by_patron_item_or_document(
             patron["id"], filter_states=active_loan_states
-        ).execute()
-
+        )
```

Comment:
is this change correct? there is no execute/scan called.

Reply:
Yes, instead of calling it here we will call `scan` in the [serializer](https://github.com/CERNDocumentServer/cds-ils/pull/363/files#diff-c3b7becdc2b2b6fb688d80b45e1f753413d1222d51ef2dee168f3611541f281dR50)